akka-javasdk/src/main/scala/akka/javasdk/impl/client/AgentClientImpl.scala

```diff
@@ -170,4 +172,72 @@ private[javasdk] final case class AgentClientImpl(
         .asJava
 
   }
+
+  override def dynamicCall[A1, R](agentId: String): ComponentMethodRef1[A1, R] = {
```

Comment:
shouldn't that return `ComponentMethodRef1[Any, Any]`?

Reply:
I see, we can leave it 'type'. I doesn't hurt. 

But in the example above, since calling programmatically, it will make more sense to not call it dynamically.